firstvoices/backend/resources/media.py

```diff
@@ -66,3 +66,87 @@ def before_import_row(self, row, **kwargs):
     class Meta:
         model = Audio
         clean_model_instances = True
+
+
+class ImageResource(ControlledSiteContentResource):
```

Comment:
I realize the AudioResource was already using this base class, but media models are actually UncontrolledSiteContent (no visibility field) so this should get changed.

Reply:
Good catch.